ipalib/install/dnsforwarders.py

```diff
@@ -0,0 +1,157 @@
+#
+# Copyright (C) 2020  FreeIPA Contributors see COPYING for license
+#
+"""DNS forwarder and systemd-resolve1 helpers
+"""
+import ipaddress
+import logging
+import os
+import socket
+
+import dbus
+
+from ipaplatform.paths import paths
+try:
+    from ipapython.dnsutil import get_ipa_resolver
+except ImportError:
+    from dns.resolver import get_default_resolver as get_ipa_resolver
```

Comment:
I'll remove this try/except block later. It makes it easier to test new code with current IPA version on F33.